tcp-ballerina/build.gradle

```diff
@@ -118,7 +118,7 @@ task updateTomlFile {
     doLast {
         def stdlibDependentIoVersion = project.stdlibIoVersion.replace("${snapshotVersion}", "")
         def stdlibDependentLogVersion = project.stdlibLogVersion.replace("${snapshotVersion}", "")
-        def nettyVersion = project.nettyVersion.replace("${snapshotVersion}", "")
+        def nettyVersion = project.nettyVersion.split("-")[0]
```

Comment:
This is unnecessary. Netty version is already in a stable version. Why do we need to replace it anyway?

Reply:
split removed